port_ocean/core/handlers/port_app_config/api.py

```diff
@@ -15,4 +15,12 @@ class APIPortAppConfig(BasePortAppConfig):
     async def _get_port_app_config(self) -> dict[str, Any]:
         logger.info("Fetching port app config")
         integration = await self.context.port_client.get_current_integration()
-        return integration["config"]
+        config = integration["config"]
+
+        if not config:
+            logger.error(
+                "The integration port app config is empty. "
+                "Please make sure to configure your port app config in the Port API."
```

Comment:
```suggestion
                "Please make sure to configure your port app config using Port's API."
```

Reply:
@yairsimantov20 again, there is almost never a reason to use `the` before Port API